src/components/vm/overview/cpuModal.jsx

```diff
@@ -271,7 +271,7 @@ export const CPUModal = ({ vm, maxVcpu, models }) => {
             <FormGroup fieldId="sockets" label={_("Sockets")}
                        labelIcon={
                            <Popover bodyContent={_("Preferred number of sockets to expose to the guest.")}>
-                               <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                               <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230704-092041-0b751cdc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)